### PR TITLE
Replace browser versions with evergreen values

### PR DESCRIPTION
**Pull request by anthonydillon**
Replaced the supported version of browsers to be more heuristic.

### CODE REVIEW

File: coding/browser-support.md [hunk 1]
```diff
@@ -4,20 +4,20 @@ layout: base
...
 ## Full support
 
 Any browser with full support needs to be pixel perfect and all functionality should work.
 
 | Browser | Version |
 | ------- | ------- |
-| Chrome | 55 or greater |
-| Firefox | 52 or greater |
-| Safari | 11 or greater |
+| Chrome | Should work 3 major versions back |
```
Comment (nottrobin) [inline coding/browser-support.md]: Maybe we could be a tad more explicit:

```suggestion
| Chrome | Should work 3 major versions behind [stable](https://www.chromestatus.com/features/schedule) |
```

**Comment by squidsoup** [comment]
👍

File: coding/browser-support.md (hunk 1 above)
Comment (anthonydillon) [reply coding/browser-support.md]: @nottrobin done

File: coding/browser-support.md [hunk 2]
```diff
@@ -4,20 +4,20 @@ layout: base
...
-| Firefox | 52 or greater |
-| Safari | 11 or greater |
+| Chrome | Should work for the last 3 [stable versions](https://www.chromestatus.com/features/schedule) |
+| Firefox | Should work for the last 3 [stable versions](https://wiki.mozilla.org/Release_Management/Calendar) |
+| Safari | [latest version](https://developer.apple.com/safari/technology-preview/release-notes) |
 | IE | 11 or greater |
-| Edge | 15 or greater |
-| Opera | 48 or greater |
+| Edge | Latest |
+| Opera | Should work for the last 3 [stable versions](https://www.opera.com/docs/history/#history) |
```
Comment (nottrobin) [inline coding/browser-support.md]: Do we need last 3 for Opera if we only do latest for Edge or Safari? Was this decision based on stats? 'cos my understanding was that Opera tends to trail behind all the others in market share.

File: coding/browser-support.md [hunk 3]
```diff
@@ -4,20 +4,20 @@ layout: base
...
 ## Full support
 
 Any browser with full support needs to be pixel perfect and all functionality should work.
 
 | Browser | Version |
 | ------- | ------- |
-| Chrome | 55 or greater |
-| Firefox | 52 or greater |
-| Safari | 11 or greater |
+| Chrome | Should work for the last 3 [stable versions](https://www.chromestatus.com/features/schedule) |
```
Comment (nottrobin) [inline coding/browser-support.md]: Apologies, @anthonydillon, I realised this should probably have the word "major" in it.

```suggestion
| Chrome | Should work for the last 3 [stable major versions](https://www.chromestatus.com/features/schedule) |
```

File: coding/browser-support.md (hunk 2 above)
Comment (anthonydillon) [reply coding/browser-support.md]: Its based on the release cadence. Edge or Safari release much more slowly then Opera, etc.

**Comment by anthonydillon** [comment]
@nottrobin all done

**Comment by nottrobin** [comment]
okay :+1:

**Pull request merged**